backend/open_webui/routers/audio.py

```
@@ -68,27 +76,47 @@
+        codec_type = info.get("codec_type", "").lower()
+        codec_tag_string = info.get("codec_tag_string", "").lower()
+
+        if codec_name == "aac" and codec_type == "audio" and codec_tag_string == "mp4a":
+            # File is AAC/mp4a audio, recommend mp3 conversion
+            return True
+
+        # If the codec name is in the supported formats
+        if codec_name in SUPPORTED_FORMATS:
+            return False
+
         return True
```

Ruff (0.12.2)

Return the condition `codec_name not in SUPPORTED_FORMATS` directly

Replace with `return codec_name not in SUPPORTED_FORMATS`

(SIM103)